Sources/ZIPFoundation/Archive+Writing.swift

```diff
@@ -189,6 +192,15 @@ extension Archive {
 
     // MARK: - Helpers
 
+    func uniqueTemporaryDirectoryURL() -> URL {
+        if let tempDir = try? FileManager().url(for: .itemReplacementDirectory, in: .userDomainMask,
+                                                appropriateFor: self.url, create: true) {
+            return tempDir
+        }
+
+        return URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(UUID().uuidString)
```

Comment:
I use `ProcessInfo.processInfo.globallyUniqueString` instead of `UUID` at several other source locations. While `globallyUniqueString` does not produce canonical UUIDs, it returns a string that is guaranteed to be unique for a network (because it uses the hostname as salt).

Reply:
I'm happy to change this... I don't understand the rationale, however. Are we concerned about collisions in the temp directory namespace?

Reply:
I saw that you already implemented the change - Thank you. 
Using the same method for UUID generation will make it easier to catch all occurrences when refactoring unique path creation (which I plan to move into a convenience method at some point).